<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QDialog" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>519</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background: url(:/Item_Back01.jpg);
}

QMessageBox{
	color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>71</y>
     <width>232</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>210</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
	color:#ffffff;
}</string>
        </property>
        <property name="title">
         <string>Modo de Visualização</string>
        </property>
        <widget class="QRadioButton" name="rbModoJanela">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>200</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>Modo Janela  | Window Mode</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbTelaCheia">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>200</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>Tela Cheia | Full Scream</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>210</width>
          <height>105</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
	color:#ffffff;
}</string>
        </property>
        <property name="title">
         <string>Resoluções disponiveis</string>
        </property>
        <widget class="QRadioButton" name="rb640x480">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>71</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>640x480</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rb1600x1280">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>40</y>
           <width>81</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>1600x1280</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rb800x600">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>71</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>800x600</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rb1366x768">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>71</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>1366x768</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rb1024x768">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>59</y>
           <width>71</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>1024x768</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rb1280x1024">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>81</width>
           <height>17</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>17</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>1280x1024</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rb1920x1080">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>60</y>
           <width>81</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>1920x1080</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>210</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
	color:#ffffff;
}</string>
        </property>
        <property name="title">
         <string>Musica</string>
        </property>
        <widget class="QRadioButton" name="rbMusicOn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>61</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>Music on</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbMusicOff">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>71</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>Music off</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>210</width>
          <height>51</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox{
	color:#ffffff;
}</string>
        </property>
        <property name="title">
         <string>Sound</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <widget class="QRadioButton" name="rbSoundOn">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>Sound on</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="rbSoundOff">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
	color: #ffffff;
}</string>
         </property>
         <property name="text">
          <string>Sound Off</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnSalvarConfig">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>105</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	margin:0;
	border :0;
	background: url(:/btn0.jpg);
	color: white;
}
QPushButton::hover
{
	margin:0;
	border :0;
	background: url(:/btn.jpg);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancelar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>105</width>
          <height>28</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	margin:0;
	border :0;
	background: url(:/btn0.jpg);
	color: white;
}
QPushButton::hover
{
	margin:0;
	border :0;
	background: url(:/btn.jpg);
	color: white;
}</string>
        </property>
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>470</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>By : LodiBR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>25</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color:#ffffff;
}</string>
   </property>
   <property name="text">
    <string>Configurações do Cliente</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
